scripts-available/CDB_TableUtils.sql

```diff
@@ -88,6 +94,16 @@ BEGIN
       );
     END IF;
 
+    -- Manually record table updates
+    PERFORM public.CDB_TableMetadataTouch(dest_table);
+
+    -- Reenable trigger for tracking updates to after manual run
+    EXECUTE FORMAT(
+      'ALTER TABLE %I.%I ENABLE TRIGGER track_updates',
+      schema_name,
+      dest_table_name
+    );
+
```

Comment:
If any of the calls between the disable/enable trigger fails the function it seems we will be in a state where the trigger will not re-enable leading to caching issues.  We need something similar to a try/finally (ie: BEGIN/EXCEPTION) to catch this case.

Reply:
All postgres functions are run in transactions, thus any transaction safe changes are automatically rolled back on error, including disabling/enabling of triggers.  We would only need to catch exceptions in the case where we need to prevent an error from rolling back the transaction, or if we had a need for custom error handling or logging.

Reply:
ok